src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,54 @@
-export const Tabs = () => {};
+ type Tab = {
+   id: string,
+   title: string,
+   content: string
+ };
+
```

Comment:
Same here. Props type is an exception because it is used only one time to define the type of the component's props. Consider using interfaces for object types because they can be extended, which is quite common for objects. I think it is a better long term solution.